src/test/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializerTest.java

```diff
@@ -46,7 +46,7 @@ public class RecordDeserializerTest {
 
   private static final SchemaErrors.SchemaId VALID_SCHEMA_ID = new SchemaErrors.SchemaId("fake_cluster_id", 10008);
 
-  SchemaErrors.ConnectionId CONNECTION_1_ID = new SchemaErrors.ConnectionId("c1");
+  String CONNECTION_1_ID = "c1";
```

Comment:
We're considering this a constant if the name is in all caps, so let's make it final
```suggestion
  static final String CONNECTION_1_ID = "c1";
```